<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>1001</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>230</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ImageSize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>705</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin Size : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_CurrentSize">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>705</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Size :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Scale">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>705</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Remove">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>705</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove All Boxes</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cursorX">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>730</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cursorY">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>730</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y : </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>230</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_txtSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_txtSaveAs">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>710</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save As</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_format">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>710</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_saved">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>740</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Images"/>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_Label_File"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Images">
   <property name="text">
    <string>Open Images</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen_Label_File">
   <property name="text">
    <string>Open Label File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
